src/condor_filetransfer_plugins/multifile_curl_plugin.cpp

```diff
@@ -259,6 +259,14 @@ MultiFileCurlPlugin::InitializeCurlHandle(const std::string &url, const std::str
 			fprintf(stderr, "Can't setopt HEADERFUNCTION\n");
 		}
 
+        // We MUST successfully set the USERAGENT, because some URLs
+        // will 403 without one. (!!)  [https://repo.anaconda.com]
+        // ... blowing as ASSERT() here is probably not OK, though.
+        r = curl_easy_setopt( _handle, CURLOPT_USERAGENT, "multifile_curl_plugin/" "PLUGIN_VERSION" );
```

Comment:
I think that the user-agent should clearly identify that this is associated with condor. Also, since the filename is curl_plugin, I think we should drop the multifile.
```suggestion
        r = curl_easy_setopt( _handle, CURLOPT_USERAGENT, "condor_curl_plugin/" "PLUGIN_VERSION" );
```